lms/djangoapps/courseware/masquerade.py

```diff
@@ -158,7 +204,7 @@ def post(self, request, course_key_string):
                 return JsonResponse({
                     'success': False,
                     'error': _(
-                        u'There is no user with the username or email address u"{user_identifier}" '
+                        u'There is no user with the username or email address "{user_identifier}" '
```

Comment:
Not really sure why we were doing this (it also displays verbatim like this, with the `u` prefix, in the existing experience), but it looks weird :shrug:

Reply:
Nit: Add period?